src/main.cpp

```diff
@@ -1,2 +1,55 @@
+#include "unix_socket_server.hpp"
+#include <iostream>
+#include <sys/un.h>
+#include <unistd.h>
```

Comment:
We don't want to have those Linux specific headers here. We already have them in `unix_socket_server.hpp`, but there they are safe behind preprocessor so that we don't try to include them in Windows build compilation.